gdk4-macos/Gir.toml

```
@@ -36,4 +35,14 @@ name = "GdkMacos.MacosMonitor"
 status = "generate"
     [[object.function]]
     name = "get_geometry"
-    ignore = true # The function does not exists
\ No newline at end of file
+    ignore = true # The function does not exists
+
+[[object]]
+name = "GdkMacos.MacosSurface"
+status = "generate"
+    [[object.function]]
+    name = "get_native_window"
+    manual = true
+    [[object.property]]
+    name = "native"
+    ignore = true # Same as native_window
```

```suggestion
    generate = ["notify"]
```
and add the getter manually. The cocoa helper, when the future is not enabled, should also be there but giving you a pointer instead of the cocoa id type.

---

do you think we should just rename native_window to native?

---

done